st2actions/st2actions/container/base.py

```diff
@@ -67,9 +67,6 @@ def dispatch(self, liveaction_db):
         return liveaction_db.result
 
     def _do_run(self, runner, runnertype_db, action_db, liveaction_db):
-        # Finalized parameters are resolved and then rendered.
```

Comment:
Seems correct.

This change was done https://github.com/StackStorm/st2/commit/8b1d04b08b43d15cccb1671b013de91660c56568#diff-ef7b3e3b028fe99ecaed2faaad32fd11

as well but somehow got dropped on the next commit.